packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -562,9 +569,7 @@ sdk.server('https://eu.api.example.com/v14');`)
 
         parameters.body = {
           name: 'body',
-          type: paramTypes.body
-            ? this.schemas.get(paramTypes.body).tsType
-            : this.schemas.get(paramTypes.formData).tsType,
+          type: paramTypes.body ? paramTypes.body : paramTypes.formData,
```

Comment:
`paramTypes` contains an object containing the TypeScript types we've created for `body` or `formData` params. For example in our own codegen'd SDK:

```ts
/**
 * Returns the doc with this slug
 *
 * @summary Get doc
 */
getDoc(
  metadata: types.GetDocMetadataParam
): Promise<types.GetDocResponse401 | types.GetDocResponse403 | types.ErrorDocNotfound> {
  return this.core.fetch('/docs/{slug}', 'get', metadata);
}
```

` types.GetDocMetadataParam` would be what's stored in `paramTypes.metadata`.